src/main/java/com/wfector/notifier/ChestShopNotifier.java

```diff
@@ -139,13 +139,59 @@ public static void close(Connection c) {
         }
     }
 
+    public static String replaceChatColors(String input) {
+        if (input == null) {
+            return null;
+        }
+//        for (int i = 0; i < ChatColor.values().length; i++) {
+//            // the enum contains no element called "ChatColor.OBFUSCATED",
+//            // but ChatColor.values()[] does !?
+//            if (!ChatColor.values()[i].name()
+//                    .equalsIgnoreCase("obfuscated")) {
+//                String replace = "&"
+//                        + ChatColor.values()[i].getChar();
+//                input = input.replace(
+//                        replace,
+//                        ChatColor.valueOf(
+//                                ChatColor.values()[i].name().toUpperCase())
+//                                .toString());
+//            }
+//        }
+//        return input;
+        return ChatColor.translateAlternateColorCodes('&', input);
+    }
+
     /**
      * Gets a message from the config file.
-     * @param string The name of the message to get
+     * @param key The name of the message to get
      * @return The message or null if it doesn't exist
      */
-    public String getMessage(String string) {
-        return (getConfig().contains("messages." + string)) ? ChatColor.translateAlternateColorCodes('&', getConfig().getString("messages." + string)) : null;
+    public String getMessage(String key) {
+        String s = getConfig().getString("messages." + key);
+        if (s != null && !s.isEmpty()) {
+            return ChestShopNotifier.replaceChatColors(s);
+        } else {
+            return "Missing string 'messages." + key + "' in config.yml";
+        }
+    }
```

Comment:
I think we only need one getMessage method as java will just use the one below with a zero sized array if no argument is provided after the first one.